.github/workflows/auto-release-bugfix.yml

```diff
@@ -0,0 +1,173 @@
+                      # as part of the github release.
+                      # NOTE: If changes were pushed to $TARGET_BRANCH before
+                      #       we try to push this release, the push will fail and
+                      #       we'll avoid the race condition.
+                      # NOTE: This push will NOT trigger other workflows because it's
+                      #       authenticated with $GITHUB_TOKEN.  By design, events
+                      #       authenticated with this token will not cause
+                      #       recursive workflow calls.
+                      GIT_REPO=https://$GITHUB_ACTOR:$GITHUB_TOKEN@github.com/$GITHUB_REPOSITORY.git
+                      git push $GIT_REPO $TARGET_BRANCH
+
+                      RELEASE_MESSAGE_FILE=release_message.md
+                      VERSION=$BUGFIX_VERSION ./ci/release/build-release-text-from-history.sh > $RELEASE_MESSAGE_FILE
+                      cat $RELEASE_MESSAGE_FILE  # for debugging, just in case.
+
+                      # Create a release object based on the text from HISTORY.
+                      # This requires that the tag already exists in the
+                      # repository on github.
+                      # NOTE: creating a GitHub release creates the tag.
+                      #       The InVEST binaries are created in response to
+                      #       this release object being created.  See the
+                      #       upload-binaries-to-release build job in
+                      #       .github/workflows/binary-applications.yml.
+                      # NOTE: The User Access Token used must have repository
+                      #       push permissions.  If the target repo is public, then
+                      #       only the ``public_repo`` permission is needed.
+                      #       See the list/descriptions of available scopes at:
+                      #       https://developer.github.com/apps/building-oauth-apps/understanding-scopes-for-oauth-apps/#available-scopes
+                      RELEASE_URL_FILE=release_url.txt
+                      GITHUB_TOKEN=${{ secrets.USER_ACCESS_TOKEN }} hub release create \
+                          --file $RELEASE_MESSAGE_FILE \
+                          --commitish "$TARGET_BRANCH" \
+                          "$BUGFIX_VERSION" > $RELEASE_URL_FILE
+
+                      # Python wheels are created in response to the release.
+                      # Create an issue for uploading the wheels to PyPI here
+                      # so that we remember to take care of this.
+                      hub issue create \
+                          --message "Release natcap.invest $BUGFIX_VERSION on PyPI" \
+                          --message "When built, remember to upload wheels for $BUGFIX_VERSION to PyPI." \
+                          --message "Wheels will be uploaded to $(cat $RELEASE_URL_FILE)" \
+                          --labels task
```

Comment:
Cool!

Reply:
The `hub` CLI is pretty great for avoiding a lot of the heavy lifting with GitHub's various APIs!